micrometer-core/src/test/java/io/micrometer/core/instrument/config/validate/PropertyValidatorTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2025 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument.config.validate;
+
+import io.micrometer.core.instrument.config.MeterRegistryConfig;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Properties;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class PropertyValidatorTest {
```

Comment:
I would put tests here with tricky spacing (like above):
```
a=b ,c =d, e = f
```

Reply:
Done in 8db4d6e5c84b1674a406fd7926bac5574606a9b5